src/components/WebexRemoteMedia/WebexRemoteMedia.js

```diff
@@ -40,7 +64,13 @@ export default function WebexRemoteMedia({className, meetingID}) {
               <div>Connecting</div>
             </Badge>
           ) : null}
-          {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
+
+          {remoteVideo ? <video ref={videoRef} playsInline autoPlay name="remote-cam" /> : null}
+
+          {remoteShare && isLocalSharingInactive ? (
+            <video ref={shareRef} playsInline autoPlay name="remote-share" className={classNames(remoteShareClasses)} />
```

Comment:
I don't think [`name` is an attribute of `video`](https://developer.mozilla.org/en-US/docs/Web/API/Element/name) elements
```suggestion
            <video ref={shareRef} playsInline autoPlay className={classNames(remoteShareClasses)} />
```